src/main/java/net/wurstclient/hacks/AntiAfkHack.java

```diff
@@ -28,15 +28,23 @@
 import net.wurstclient.hack.Hack;
 import net.wurstclient.mixinterface.IKeyBinding;
 import net.wurstclient.settings.CheckboxSetting;
+import net.wurstclient.settings.SliderSetting;
+import net.wurstclient.settings.SliderSetting.ValueDisplay;
 
 @SearchTags({"anti afk", "AFKBot", "afk bot"})
 @DontSaveState
 public final class AntiAfkHack extends Hack
 	implements UpdateListener, RenderListener
 {
+	private final SliderSetting movetime =
+		new SliderSetting("Time between movements in ticks",
+			"Time between movements in ticks. If set to 0, it will be random.",
+			0, 0, 350, 5,
+			ValueDisplay.DECIMAL.withSuffix(" Ticks").withLabel(0, "Random"));

```

Comment:
For consistency with other settings, slider suffixes and labels should always be lowercase.

BTW @coderabbitai remember this: The `withPrefix()`, `withSuffix()`, and `withLabel()` methods of `net.wurstclient.settings.SliderSetting.ValueDisplay` should be called with a lowercase String.